.github/workflows/ci.yml

```diff
@@ -8,6 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [10, 12, 14]
```

Comment:
Node 16 is now LTS, maybe include that as well? Not sure what the policy is from the main repo on supported Node versions